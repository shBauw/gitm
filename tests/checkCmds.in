help
place A1
place A1
view
history
place A20
place Z9
place A555
place 44
place  A11
term
