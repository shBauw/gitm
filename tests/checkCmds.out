Invalid!
Occupied coordinate
M3,..........................................xxxxxxx
A1
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid!
Invalid!
